### ISSUE TITLE
Pagination

### ISSUE THREAD

**Issue by vyrzdev** [sscheetz/etsy-python2#10]
As a suggestion, you may want to mention that the count variable returned from Etsy API that is essential for proper pagination is stored on the Etsy instance as `EtsyInstance.count` in the readme

It also may be a better idea not to extract the results in the request code, and simply have the user unpack that later, as the metadata returned by Etsy may have value.

Storing the count variable on the instance also prevents making multiple paginated requests simultaneously, which will limit the optimisation that can be done.

**Comment by sscheetz** [comment]
Good suggestions. I'm open to PRs of course, but I'll try to take a look sometime this week or next. A bit busy with work atm.

**Comment by sscheetz** [comment]
yeah sorry I don't have time to do it at the current time. If someone wants to pick this up I will review the PR.
